### ISSUE TITLE
FE boundary for Germany

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#194]
## Description
See #193 
Currently the map shows every region, we want only germany displayed

## Acceptance criteria
* Permanent boundary for germany
* Visibility for other regions is reduced (e.g.: white background)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
works in satellite view, more compilcated with masking mechanism

**Comment by op-hub** [comment]
Closed for now: Effort exceeds benefit, concerning time restraints
